src/lib/viewers/error/PreviewErrorViewer.js

```diff
@@ -77,8 +76,15 @@ class PreviewErrorViewer extends BaseViewer {
             }
         }
 
+        /* eslint-disable no-param-reassign */
+        err = err instanceof Error ? err : new Error(__('error_default'));
+        /* eslint-enable no-param-reassign */
+
+        // If there is no display message fallback to the message from above
+        const displayMessage = err.displayMessage || err.message;
```

Comment:
@tonyjin just to make sure we're on the same page here.

Display message will be either:
1. display message from passed in error
2. message of locally created error (default)

Log message will be:
1. message from passed in error (hopefully useful)
2. message of locally created error (default)
3. display message from passed in error (case with display message but no message)